src/main/java/uk/gov/hmcts/darts/audio/component/impl/OutboundFileProcessorImpl.java

```diff
@@ -157,7 +195,7 @@ private List<List<AudioFileInfo>> concatenateByChannelWithGaps(List<AudioFileInf
                 audioFileInfosForChannel,
                 allowableAudioGap
```

Comment:
The property `darts.audio.allowable_audio_gap_duration` has underscores whereas the others are hyphenated.
I think it should be made hyphenated and added to the `AudioConfigurationProperties` class we have already. So we can get rid of the `@Value` annotation on line 35 OutboundFileProcessorImpl#allowableAudioGap?

Reply:
Another observation `AudioTransformationServiceImpl#handlePlaybacks` - I think this public method should be refactored (pushed up) into the private handlePlaybacks as it's a bit redundant and shouldn't be public access anyway?  IntelliJ shows a whinge on Line 350, so I think it's probably worth another a look to tidyup playback a bit.